docs/UserGuide.md

```diff
@@ -550,16 +548,20 @@ Format: `exit`
 
 ### 7.1. Appendix A: Acceptable values for parameters
 
-Acceptable values for `n/NAME`:
+>:information_source: **Definitions** <br />
+>Word: Any letters, numbers or special characters of length 1 or more (non-blank). <br />
+>Alphanumeric word: Any letters numbers of length 1 or more (non-blank). <br />
+
+<br />
```

Comment:
This looks great! May want to sync the format of the color tips with @tllshan though (#266)?

Reply:
Format synced!